CITATION.cff

```diff
@@ -0,0 +1,36 @@
+# YAML 1.2
+---
+abstract: "Pydantic is the most widely used data validation library for Python."
+authors:
+  - family-names: Colvin
+    given-names: Samuel
```

Comment:
Could you add all authors listed here (I think you're missing Alex): https://github.com/pydantic/pydantic/blob/main/pyproject.toml

Reply:
There's potentially other info that you could pull from that file as well, looks good overall! Thanks!